pkg/helm/run.go

```diff
@@ -53,9 +53,13 @@ func Run(flags *hoflags.HelmOperatorFlags) error {
 	namespace, found := os.LookupEnv(k8sutil.WatchNamespaceEnvVar)
 	log = log.WithValues("Namespace", namespace)
 	if found {
-		log.Info("Watching single namespace.")
+		if namespace == metav1.NamespaceAll {
+			log.Info("Watching all namespaces.")
+		} else {
+			log.Info("Watching single namespace.")
```

Comment:
WDYT should we include the name of the namespace?

Reply:
I agree! We do that in line 54.

Reply:
ah missed that! :) 👍